### PR TITLE
feature: pouch run in terminal default connects to container's stdout

### PR DESCRIPTION
**Pull request by HusterWan**
Signed-off-by: HusterWan <zirenwan@gmail.com>

**1.Describe what this PR did**
As the title says

**2.Does this pull request fix one issue?** 

fixes https://github.com/alibaba/pouch/issues/491

**3.Describe how you did it**

**4.Describe how to verify it**
```
[root@docker-registry github.com]# pouch run -e aaa=bbb registry.hub.docker.com/library/busybox:latest env
PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
aaa=bbb
TERM=xterm
HOME=/root
```

```
[root@docker-registry github.com]# pouch run -d -e aaa=bbb registry.hub.docker.com/library/busybox:latest env
90719b5f9a455b3314a49e72e3ecb9962f215e0f90153aa8911882acf2ba2c84
```

**5.Special notes for reviews**

### CODE REVIEW

**Comment by allencloud** [comment]
Could you add a test case in integration test? @HusterWan 
I think that could be better for code adding.

**Comment by HusterWan** [comment]
Updated, @allencloud

File: test/cli_run_test.go [hunk 1]
```diff
@@ -34,14 +34,26 @@ func (suite *PouchRunSuite) TearDownTest(c *check.C) {
 func (suite *PouchRunSuite) TestRun(c *check.C) {
 	name := "test-run"
 
-	_ = command.PouchRun("run", "--name", name, busyboxImage).Assert(c, icmd.Success)
+	_ = command.PouchRun("run", "-d", "--name", name, busyboxImage).Assert(c, icmd.Success)
```
Comment (0x04C2) [inline test/cli_run_test.go]: It seems that we don't need `_ = ` here. :)

**Comment by allencloud** [comment]
LGTM, Thanks a lot for your work.

**Pull request merged**